scripts/publish-native.js

```diff
@@ -15,36 +15,33 @@ const cwd = process.cwd()
 
     // Copy binaries to package folders, update version, and publish
     let nativePackagesDir = path.join(cwd, 'packages/next/build/swc/npm')
-    let nativePackages = await readdir(nativePackagesDir)
-    for (let nativePackage of nativePackages) {
-      if (nativePackage === '.gitignore') {
-        continue
-      }
-      let binaryName = `next-swc.${nativePackage.substr(9)}.node`
+    let platforms = await (await readdir(nativePackagesDir)).filter(
```

Comment:
I think the outside await is not necessary here since filter() is not async

Reply:
oh yeah, not sure what happened there